peerconnection.go

```diff
@@ -797,17 +798,17 @@ func (pc *PeerConnection) startReceiver(incoming trackDetails, receiver *RTPRece
 		return
 	}
 
-	codec, err := pc.api.mediaEngine.getCodec(receiver.Track().PayloadType())
-	if err != nil {
-		pc.log.Warnf("no codec could be found for payloadType %d", receiver.Track().PayloadType())
+	codecs := pc.api.mediaEngine.GetCodecsByKind(incoming.kind)
```

Comment:
This may cause misselection of the codec when multiple codecs are registered for same kind. By default, 4 audio codecs and 3 video codecs are registered, and an other peer may select any ones from them. For example, if the other peer selects the second codec, this logic still select the first one.
https://github.com/pion/webrtc/blob/4220088d782ea9446907aae655c58822e35e9d2c/mediaengine.go#L43-L52

I think storing the first raw RTP packet fetched by `determinePayloadType()` to `Track` and return from `Track.Read()` is better. (It would be better to have a unit test if possible.)